backend.tex

```diff
@@ -1,24 +1,38 @@
 \section{Backend Services\label{sec:backend}}
 
-The user-facing aspects of the LSST Science Platform will be built on top of a number of backend services that can roughly be divided into three categories: \textbf{database services}, \textbf{file services}, and \textbf{batch computing} services (bottom row of Figure~\ref{fig:layeredLSP}). The details of these services are described in the Data Management Design Document (\citeds{LDM-148}) and other associated documents; here, we only provide the high-level guidance as to the capabilities which these services will need to expose to the user (through the three aspects).
+The user-facing Aspects of the LSST Science Platform will be built on top of a number of backend services that can roughly be divided into three categories: \textbf{database services}, \textbf{file services}, and \textbf{batch computing} services (bottom row of Figure~\ref{fig:layeredLSP}).
+The details of these services are described in the Data Management Design Document (\citeds{LDM-148}) and other associated documents; here, we only provide the high-level guidance as to the capabilities which these services will need to expose to the user (through the three Aspects).
```

Comment:
I'm not sure what the full set of document that would be appropriate to mention actually is.  Most likely the original intent was to just indirect through LDM-148 in order to avoid making a project-level change-controlled document dependent on a whole list of DM-level documents.  I think that was probably a good idea.

Reply:
So, to be clear, I'm not doing anything about this right now and will leave this "conversation unresolved" in the PR.